src/components/SearchBar.tsx

```diff
@@ -12,80 +12,73 @@ import SearchBarNativeComponent, {
 } from '../fabric/SearchBarNativeComponent';
 
 export const NativeSearchBar: React.ComponentType<SearchBarProps> &
-  typeof NativeSearchBarCommands = SearchBarNativeComponent as any;
+  typeof NativeSearchBarCommands =
+  SearchBarNativeComponent as unknown as React.ComponentType<SearchBarProps> &
+    SearchBarCommandsType;
 export const NativeSearchBarCommands: SearchBarCommandsType =
-  SearchBarNativeCommands as any;
+  SearchBarNativeCommands as SearchBarCommandsType;
+
+type ViewRef = React.ElementRef<typeof NativeSearchBar>;
```

Comment:
```suggestion
type NativeSearchBarRef = React.ElementRef<typeof NativeSearchBar>;
```

Let's use more descriptive name, but this alias is very good idea.

I'm [nitpicking](https://dictionary.cambridge.org/pl/dictionary/english/nitpick) here hard, because our attention won't be back to this component for some time most likely 😄

Reply:
https://github.com/software-mansion/react-native-screens/pull/2132/commits/33cc656967c6ab70262008ec3f79ac2c9ad1a550